bin/exec.go

```diff
@@ -59,8 +59,11 @@ func main() {
 		experiment := utils.NewExperimentDetails()
 		experiment.SetValueFromChaosEngine(engineDetails, i)
 		experiment.SetValueFromChaosExperiment(clients)
-		experiment.SetENV(engineDetails, clients)
-
+		err := experiment.SetENV(&engineDetails, clients)
+		if err != nil {
+			log.Infof("Unable to set ENV %v", err)
+			break
```

Comment:
we seem to do this in case of other attributes as well.. is this what we want to do (break) ? @rahulchheda @chandankumar4 - is this to ensure ability to run through other experiments in the list, if applicable?

Reply:
This break will terminate the `engineDetails.Experiments` loop. It means if one experiment is failed then it will terminate the execution of other remaining experiments also